curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/64faca774fd9fd74bc084cc9.md

```diff
@@ -66,10 +66,10 @@ assert.match(code, /const\s*formInputsContainValues\s*=\s*titleInput\.value\s*\|
         <form method="dialog">
           <p class="discard-message-text">Discard unsaved changes?</p>
           <div class="confirm-close-dialog-btn-container">
-            <button id="cancel-btn" class="btn" aria-label="cancel">
+            <button id="cancel-btn" class="btn">
               Cancel
             </button>
-            <button id="discard-btn" class="btn" aria-label="discard">
+            <button id="discard-btn" class="btn">
```

Comment:
I need clarity on why the `aria-label="cancel"` was removed instead of replacing it with `aria-label="Cancel Button"`.

Reply:
Because the accessible name for this button is provided by the text "Cancel" between the opening and closing `button` tags. It is not necessary to change the accessible name to "Cancel Button" using `aria-label` for a screen reader user's sake as their screen reader will already announce that it is a button. If we did that then they would hear "Button Cancel Button" which is redundant and unnecessary.

Reply:
> Because the accessible name for this button is provided by the text "Cancel" between the opening and closing `button` tags. It is not necessary to change the accessible name to "Cancel Button" using `aria-label` for a screen reader user's sake as their screen reader will already announce that it is a button. If we did that then they would hear "Button Cancel Button" which is redundant and unnecessary.

This makes sense! Thank you, @bbsmooth, for your feedback.